### ISSUE TITLE
Adapter question: integration test and release

### ISSUE THREAD

**Issue by katzhang-ias** [prebid/Prebid.js#2006]
## Type of issue

question

## Description

Hi there! I'm working on a new bidder adapter for Integral Ad Science. Our team has been reading the [adapter documentation](http://prebid.org/dev-docs/bidder-adaptor.html) and yet still got a couple of questions:

1. If there is any suggested way to run integration/functional tests on the adapter module;
2. what the release process is for adapters (does prebid.js host the adapter code?)

Thanks in advance!

**Comment by harpere** [comment]
Hi,

1. After you've built prebid with your new adapter, this `hello_world.html` example page is a good way to test your adapter:  https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/hello_world.html
2. Once your adapter is merged into master, it will be included in the next release (new releases are tagged usually every-other Tuesday).   At that point your adapter will be available to anyone who wants to include it in a new build.  Prebid does not host the built Prebid.js library.  A third party (ex. a Publisher, you) would have to host it.  If a Publisher already using Prebid wants to use your adapter they could have whomever is hosting their built prebid.js lib to include your adapter in their next build.

Does this answer your questions?

**Comment by katzhang-ias** [comment]
Thank you @harpere for the answer ! Closing the issue.
